README.md

```diff
@@ -92,6 +106,9 @@ spec:
 * If needed, you can use [ConfigMap](http://kubernetes.io/docs/user-guide/configmap/)
 to overwrite the `config/`.
 
+## Start Standalone
+`node-problem-detector -apiserver=http://APISERVER_IP:APISERVER_PORT?inClusterConfig=false&apiVersion=v1`
```

Comment:
done.
@Random-Liu Examples about using `serviceAccount` and `kubeconfig` is out of mine knowledge now. :(.

Maybe others can do this. :)